libraries/datahandler/src/main/thrift/sw360.thrift

```diff
@@ -66,13 +66,14 @@ enum ModerationState {
 
 enum ClearingRequestState {
     NEW = 0,
-    ACCEPTED = 1,
-    REJECTED = 2,
-    IN_QUEUE = 3,
-    IN_PROGRESS = 4,
-    CLOSED = 5,
-    AWAITING_RESPONSE = 6
-    ON_HOLD = 7
+    SANITY_CHECK = 1,
+    ACCEPTED = 2,
+    REJECTED = 3,
+    IN_QUEUE = 4,
+    IN_PROGRESS = 5,
+    CLOSED = 6,
+    AWAITING_RESPONSE = 7,
+    ON_HOLD = 8
```

Comment:
I suggest not to change the **`enum`** values, and keep the new entry as `SANITY_CHECK = 8`.
It might create issue if these values are mapped / inserted to the database instead of keys. if it's mapped, then you might need to write the migration script or else this change should be good.